.github/workflows/ci.yml

```diff
@@ -68,6 +68,8 @@ jobs:
         run: make license-check
       - name: Check requirements
         run: make check-req
+      - name: Make generate
+        run: make generate
```

Comment:
Why suddenly need this? But didn't before?

Reply:
```suggestion
```